src/libs/actions/Policy.js

```diff
@@ -403,6 +421,67 @@ function hideWorkspaceAlertMessage(policyID) {
     Onyx.merge(`${ONYXKEYS.COLLECTION.POLICY}${policyID}`, {alertMessage: ''});
 }
 
+/**
+ * @param {String} policyID
+ * @param {Object} values
+ */
+function setCustomUnit(policyID, values) {
+    const payload = {
```

Comment:
No real need for this variable, just define this object inside the call to `API.Policy_CustomUnit_Update`.